content/sensu-go/5.17/reference/checks.md

```diff
@@ -528,7 +528,7 @@ example      | {{< highlight shell >}}"namespace": "production"{{< /highlight >}
 
 | labels     |      |
 -------------|------
-description  | Custom attributes to include with event data that you can access with [event filters][27].<br><br>In contrast to annotations, you can use labels to create meaningful collections that you can select with [API response filtering][54] and [sensuctl response filtering][55]. Overusing labels can affect Sensu's internal performance, so we recommend moving complex, non-identifying metadata to annotations.
+description  | Custom attributes to include with event data that you can use for response and dashboard filtering.<br><br>If you include labels in your event data, you can filter [API responses][54], [sensuctl responses][55], and [dashboard views][58] based on them. In other words, labels allow you to create meaningful groupings for your data.<br><br>Limit labels to metadata you need to use for response filtering. For complex, non-identifying metadata that you will *not* need to use in response filtering, use annotations rather than labels.
```

Comment:
```suggestion
description  | Custom attributes to include with event data that you can use for response and dashboard view filtering.<br><br>If you include labels in your event data, you can filter [API responses][54], [sensuctl responses][55], and [dashboard views][58] based on them. In other words, labels allow you to create meaningful groupings for your data.<br><br>Limit labels to metadata you need to use for response filtering. For complex, non-identifying metadata that you will *not* need to use in response filtering, use annotations rather than labels.
```